src/pages/messaging/agent-chat-api/v3.4/rtm-reference/index.mdx

```diff
@@ -1320,9 +1322,10 @@ Deactivates a chat by closing the currently open thread. Sending messages to thi
 
 #### Request
 
-| Parameter | Required | Data type |     |
-| --------- | -------- | --------- | --- |
-| `id`      | Yes      | `string`  |     |
+| Parameter                   | Required | Data type | Notes |
```

Comment:
Guess we don't need Notes if it's empty, but shouldn't we add a note for `ignore_requester_presence`?

Reply:
The empty column looked out of place to me, and adding Notes there (even if there are no notes) is more consistent with other tables than removing the column.
As to the note, something like this?

If `true`, the requester doesn't need to be present on the list of chat users; default `false`

Reply:
Hm, I'm not sure if we should prioritize consistency instead of simplicity in this case. But since a note is needed anyway, we don't have to solve the dilemma at this very moment :P 

The note's fine, just remember about the final full stop and about adding to other tables with this param